.github/workflows/browser-compatibility-test.yml

```diff
@@ -0,0 +1,85 @@
+  SAUCE_ACCESS_KEY: ${{secrets.SAUCE_ACCESS_KEY}}
+
+jobs:
+  integ-video:
+    name: Browser Compatibility Test
+    runs-on: ubuntu-latest
+
+    steps:
+      - name: Checkout Package
+        uses: actions/checkout@v2
+        with:
+          fetch-depth: 0
+      - name: Pack the Chime SDK and install the tarball into the Demo
+        run: |
+          current_version=$(.github/script/get-current-version)
+          echo "Packing current version:" $current-version
+          npm run build
+          npm pack
+          cd demos/browser
+          npm uninstall amazon-chime-sdk-js
+          npm install ../../amazon-chime-sdk-js-$current_version.tgz
+      - name: Create a Job ID
+        id: create-job-id
+        uses: filipstefansson/uuid-action@ce29ebbb0981ac2448c2e406e848bfaa30ddf04c
+      - name: Set JOB_ID Env Variable
+        run: echo "JOB_ID=${{ steps.create-job-id.outputs.uuid }}" >> $GITHUB_ENV
+      - name: Echo Job ID
+        run: echo "${{ steps.create-job-id.outputs.uuid }}"
+      - name: Configure AWS Credentials
+        uses: aws-actions/configure-aws-credentials@v1
+        with:
+          aws-access-key-id: ${{ secrets.AWS_ACCESS_KEY }}
+          aws-secret-access-key: ${{ secrets.AWS_SECRET_ACCESS_KEY }}
+          aws-region: us-east-1
+      - name: Setup Sauce Connect
+        uses: saucelabs/sauce-connect-action@v1
+        with:
+          username: ${{ secrets.SAUCE_USERNAME }}
+          accessKey: ${{ secrets.SAUCE_ACCESS_KEY }}
+          noSSLBumpDomains: all
+          tunnelIdentifier: ${{ steps.create-job-id.outputs.uuid }}
+      - name: Setup Node.js - 14.x
+        uses: actions/setup-node@v1
+        with:
+          node-version: 14.x
+      - name: Install Kite
+        run: integration/js/script/install-kite
+      - name: Clean Install
+        run: npm ci
+      - name: Run Audio Compatibility Test
+        run: npm run test:browser-compatibility-audio
+      - name: Run Video Compatibility Test
+        run: npm run test:browser-compatibility-video
+      - name: Run Video Test App Compatibility Test
+        run: npm run test:browser-compatibility-video-test-app
+      - name: Run Content Share Compatibility Test
+        run: npm run test:browser-compatibility-content-share
+      - name: Run Data Message Compatibility Test
+        run: npm run test:browser-compatibility-data-message
+      - name: Run Meeting Readiness Checker Compatibility Test
+        run: npm run test:browser-compatibility-meeting-readiness-checker
+      - name: Run Messaging Compatibility Test
+        run: npm run test:browser-compatibility-messaging
+      - name: Run Transcription Compatibility Test
+        run: npm run test:browser-compatibility-transcription
+      - name: Run Background Blur Compatibility Test
+        run: npm run test:browser-compatibility-background-blur
+      - name: Run Voice Focus Compatibility Test
+        run: npm run test:browser-compatibility-voice-focus
```

Comment:
Steps run sequentially - jobs run in parallel. Are you sure you want these all sequential?

Reply:
We run it overnight and once per day. No reason to run it in parallel. Maybe when we have too many tests.